client/src/App.jsx

```diff
@@ -33,6 +33,7 @@ function App() {
                   <Switch>
                     <Route exact path="/" component={Landing} />
                     <Route path="/user/:userId/items" component={ItemList} />
+                    <Route path="/OAuthLink" component={OAuthLink} />
```

Comment:
By convention on the web, and in this app, routes should be lowercase and only use hyphens to separate words or terms. You will need to update references to this url, but i left suggestions where needed (i think)

```suggestion
                    <Route path="/oauth-link" component={OAuthLink} />
```